clients/ops/admin-ui/src/features/datastore-connections/add-connection/manual/DSRCustomizationForm.tsx

```diff
@@ -0,0 +1,163 @@
+import { Box, HStack, Text, VStack } from "@fidesui/react";
+import { useAlert } from "common/hooks";
+import { TrashCanSolidIcon } from "common/Icon";
+import { FieldArray, Form, Formik } from "formik";
+import { useRouter } from "next/router";
+import React from "react";
+import * as Yup from "yup";
+
+import { STEPS } from "../constants";
+import CustomInput from "../forms/CustomInput";
+import { ButtonGroup as ManualButtonGroup } from "./ButtonGroup";
+import { Field } from "./types";
+
+type DSRCustomizationFormProps = {
+  data: Field[];
+  isSubmitting: boolean;
+  onSaveClick: (values: any, actions: any) => void;
+};
+
+const DSRCustomizationForm: React.FC<DSRCustomizationFormProps> = ({
+  data = [],
+  isSubmitting = false,
+  onSaveClick,
+}) => {
+  const router = useRouter();
+  const { errorAlert } = useAlert();
+
+  const handleCancel = () => {
+    router.push(STEPS[1].href);
+  };
+
+  const handleSubmit = (values: any, actions: any) => {
+    const uniqueValues = new Set(values.fields.map((f: Field) => f.pii_field));
+    if (uniqueValues.size < values.fields.length) {
+      errorAlert("PII Field must be unique");
+      return;
+    }
+    onSaveClick(values, actions);
+  };
+
+  return (
+    <Formik
+      initialValues={{
+        fields:
+          data.length > 0
+            ? data
+            : ([
+                {
+                  pii_field: "",
+                  dsr_package_label: "",
+                },
+              ] as Field[]),
+      }}
+      onSubmit={handleSubmit}
+      validateOnBlur={false}
```

Comment:
It seems like if I have text in the DSR Package Label but change focus, I still have an error showing that it is required unless I hit enter.   I might recheck if there's text when focus changes.

![Screen Shot 2022-09-14 at 12 13 35 PM](https://user-images.githubusercontent.com/9755598/190219543-7e6e6930-379f-46d4-99fe-812d597aa531.png)